### PR TITLE
404 page UI

### LINKED ISSUE #625: Improve 404 page UI
## Requirements 

- **Feel free to work as per your ideas. We do not have any specifications for this.**
- You can take inspirations from platforms like Dribble, Codepen, etc but **DO NOT COPY PASTE THE EXACT THING**
- Make sure the routing is not messed up
- Try out if the 404 functionality is working before giving the PR
- The issue is self-explanatory given one has read the above points. Looking forward to your design tweaks!

## General PR Guidelines:

- The title of the PR is short and meaningful
- You refer to the respective issue in the PR description ``Resolves # issue_number``
- Attach screenshot(s)/clip of the result of your changes
- (In bullet points) describe your changes

### PR DESCRIPTION
**Pull request by vedikaag99**
Closes #625

### Screenshot for reference:

https://user-images.githubusercontent.com/67699840/111788448-19b3c700-88e6-11eb-9024-a2a749cb89af.mov

### CODE REVIEW

File: src/pages/404/notFound.jsx [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import React, { useEffect } from "react";
 import "./squiggly-animation.css";
 import style from "./notFound.module.css";
-import sadImg from "./../../assets/images/404/sad404.svg";
+import error1 from "./../../assets/images/404/404.gif";
```
Comment (iamdarshshah) [inline src/pages/404/notFound.jsx:4]: Rename it to something meaningful ex: `pageNotFound` or something similar.
```suggestion
import pageNotFound from "./../../assets/images/404/404.gif";
```

**Comment by vedikaag99** [comment]
@iamdarshshah I have made the suggested changes.

File: src/pages/404/notFound.module.css [hunk 2]
```diff
@@ -26,86 +25,55 @@ p {
   margin: 0;
 }
 
-.sadImg {
+.error{
```
Comment (iamdarshshah) [inline src/pages/404/notFound.module.css:28]: @vedikaag99 Please update the className accordingly

Comment (vedikaag99) [reply src/pages/404/notFound.module.css]: done

**Review by iamdarshshah — APPROVED** [review]
GIF looks good! Nice work 🎉

**Comment by anushbhatia** [comment]
Great Work with the GIF but creating a microaimation for the same would be wonderful nevertheless GG🙌.

**Comment by smaranjitghose** [comment]
Nice work @vedikaag99 !

**Pull request merged**